Imshk.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Shashank Dwivedi",
+  "type": "personal",
+  "bio": "Student, Full stack developer, Blogger",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Shashankoffice",
+      "icon": "FaTwitter"
+  },
+    {"name": "Follow me on Github ",
+      "url": "https://github.com/Imshk",
+      "icon": "FaGithub"
+    },
+    {"name": "Follow me on linkedIn ",
+      "url": "https://www.linkedin.com/in/shashankoffice/",
+      "icon": "FalinkedIn"
```

You're probably using an `invalid` icon here. It should be `FaLinkedin`.
```suggestion
      "icon": "FaLinkedin"
```